tasks/mssql_conf_setting.yml

```diff
@@ -0,0 +1,23 @@
+- name: Check if the {{ __mssql_conf_setting }} setting is enabled
+  command: >-
```

Comment:
Does `command` module work with `|`? I'd think it should be `shell`.

Reply:
It worked for me, but Rich said the same, I will change this to shell.

Reply:
I got rid of piping as Rich suggested.